.github/workflows/sycl-linux-build.yml

```diff
@@ -188,6 +197,9 @@ jobs:
       run: |
         # TODO consider moving this to Dockerfile.
         export LD_LIBRARY_PATH=/usr/local/cuda/compat/:/usr/local/cuda/lib64:$LD_LIBRARY_PATH
+        if [[ ${{ inputs.cc }} == 'icx' ]] || [[ ${{ inputs.cxx }} == 'icpx' ]]; then
+           export LIT_XFAIL="regression/host_tanpi_double_accuracy.cpp"
```

Comment:
Same. Let's add a comment if XFAIL is expected (due to use of oneAPI). If XFAIL is unexpected, perhaps we should open a bug report.

Reply:
i think its expected so will add comment, thanks

Reply:
removed in latest commits, thx